app/personalised-prevention-platform/2025/10/presenting-options/index.md

```diff
@@ -0,0 +1,206 @@
+
+* minimal and highly atomic content
+* zero imagery
+
+to:
+
+* looser content retaining a strong structure
+* minimal imagery
+
+{% from "nhsuk/components/images/macro.njk" import image as nhsukImage %}
+{{ nhsukImage({
+  classes: "app-media--full-width",
+  src: "service-detail-iterations@2x.png",
+  alt: "Screen grabs displaying 2 iterations of a details page from left to right.",
+  caption: "2 iterations of our details page"
+}) }}
+
+## What we learnt
+
+### Blend national and local
+
+Since [discovery](/personalised-prevention-platform/2025/03/discovery-summary/) we’ve continuously proved that presenting a blend of national and local has real value for people. Throughout our sessions people asked if the options were real (they all were), and then make notes to look them up afterwards.
+
+“National” and “local” are false distinctions, very visible to us, as we operate within organisational structures.
+
+But where a thing “comes from” is irrelevant. You can be interested in Active 10, interested in your local Parkrun, and interested in the public gym in your local park.
+
+### Strike a balance between needs and wants
+
+Earlier onboarding prototypes included goal and priority setting segments, along with asking about barriers &ndash; things that could get in the way.
+
+We tested removing these segments, instead asking a series of questions directly mapped to filters in the results listing, for example:
+
+![A question page asking 'how do you like to be taught or coached?' alongside a column of filters displaying the same](example-filter-question@2x.png 'Onboarding questions mapped to filters')
+
+This reductive approach led to some good evidence.
+
+Matching onboarding questions to listing view filters didn’t seem to cause any overt issues, and generally people understood how to change the initial set of options presented.
+
+However, reliance on preference alone means any recommendation aspect is negated. We’re in pure service finder territory, and we’ve left no room for the unexpected or left-field that might spark engagement.
+
+Some people told us they were basing their preferences on past experience &ndash; but that past experience was rooted in activities that had lapsed. So arguably we have a risk of presenting similar or identical options to those that may have failed the user in the past.
+
+We’ve proven that we need to become more opinionated in the options we present. These options must be mapped to a user’s declared goals, priorities, barriers, and preferences, but also weighted by “systemic” opinion.
+
+### Continuously assess the relationship between volume, variety, and granularity
+
+Generally people understood the connection between our onboarding questions, the initial set of options, and the associated filters.
+
+With 18 services in our prototype API, one or two users combined preferences or filters that led to zero results. This led to immediate disengagement and in real life, dropoff.
```

Comment:
Could you add something about what you might do to prevent that in the future, or will that be a inevitable part of this?

Reply:
This is actually mentioned a bit further down - layer on more localised services and you basically get a "no dead ends" ability